plugins/services/src/js/structs/Framework.js

```diff
@@ -47,7 +49,7 @@ module.exports = class Framework extends Application {
       }
       if (task.statuses != null) {
         return task.statuses.reduce(function(memo, status) {
-          if (status.healthy) {
+          if (status.healthy || (isSDK && status.healthy === undefined)) {
```

Comment:
This field can either be `true`, `false` or `undefined` because the property doesn't exist. I do agree that is SHOULD be a defined type - just like all our primitive types.
![](https://cl.ly/461b3P3m2X0W/Image%202017-08-03%20at%204.48.48%20PM.png)